cafebabel/templates/articles/macros.html

```diff
@@ -15,3 +15,21 @@
     Reading time: {{ article.body|reading_time }} min
   </p>
 {%- endmacro %}
+
+{%- macro tags_fields(article=None) -%}
+  <div class=tags>
+    <label for=tag-1>Tags</label>
+    <ul class=tags-list>
```

Comment:
Here we use `tags-list` for style and `.tags ul` on the JS side. We have to be consistent @pointbar @cafebabel-webmaster